BoxContentSDKSampleApp/Makefile

```diff
@@ -0,0 +1,30 @@
+CARTHAGE := $(shell command -v carthage 2> /dev/null)
+
+.PHONY: help
+
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+bootstrap: check_carthage ## Build the reference app dependency framework(s) to cartfile.resolved
+	# Make sure submodules handled by Carthage are up to date (as saved in Cartfile.resolved)
+	$(CARTHAGE) bootstrap --project-directory . --platform iOS
```

Comment:
No real need to have `--project-directory .` in any of these commands.

Reply:
done